CondTools/DT/plugins/DTKeyedConfigDBDump.cc

```diff
@@ -46,28 +67,28 @@ DTKeyedConfigDBDump::~DTKeyedConfigDBDump() {}
 //--------------
 void DTKeyedConfigDBDump::beginJob() { return; }
 
-void DTKeyedConfigDBDump::analyze(const edm::Event& e, const edm::EventSetup& c) {
+void DTKeyedConfigDBDump::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup) {
   edm::eventsetup::EventSetupRecordKey recordKey(
       edm::eventsetup::EventSetupRecordKey::TypeTag::findType("DTKeyedConfigListRcd"));
   if (recordKey.type() == edm::eventsetup::EventSetupRecordKey::TypeTag()) {
     //record not found
-    std::cout << "Record \"DTKeyedConfigListRcd "
-              << "\" does not exist " << std::endl;
+    edm::LogWarning("DTKeyedConfigDBDump") << "Record \"DTKeyedConfigListRcd "
+                                           << "\" does not exist " << std::endl;
   }
-  edm::ESHandle<cond::persistency::KeyList> klh;
-  std::cout << "got eshandle" << std::endl;
-  c.get<DTKeyedConfigListRcd>().get(klh);
-  std::cout << "got context" << std::endl;
-  cond::persistency::KeyList const& kl = *klh.product();
-  cond::persistency::KeyList const* kp = &kl;
-  std::cout << "now load and get" << std::endl;
+  //edm::ESHandle<cond::persistency::KeyList> klh;
```

Comment:
Is this commented out code needed? If so, please add a line of comment which specifies what it is intended for. Otherwise, just remove it.

Reply:
Done